topics/reactiveui/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Paul Betts
+display_name: ReactiveUI
+github_url: https://github.com/reactiveui/reactiveui
+logo: reactiveui.png
+released: 2009
+short_description: ReactiveUI is a composable, cross-platform model-view-viewmodel framework.
```

Comment:
What do you think about also mentioning .NET here?

Reply:
Hum.. you are right, I'll do that as well.